### PR TITLE
Add vue-multiselect and vuelidate

### PR DESCRIPTION
**Pull request by shentao**

### CODE REVIEW

File: PACKAGES.md [hunk 1]
```diff
@@ -13,7 +13,12 @@
...
 - [v-tooltip](https://github.com/Akryum/vue-tooltip), vue:2.0, links:[demo](https://akryum.github.io/vue-tooltip/), status:dev
+- [vue-multiselect](https://github.com/monterail/vue-multiselect), vue:1.0|2.0, links:[demo](http://monterail.github.io/vue-multiselect/), status:stable
 
 # Data
 
 - [vue-apollo](https://github.com/Akryum/vue-apollo/), vue:1.0|2.0, links:[howto](https://dev-blog.apollodata.com/use-apollo-in-your-vuejs-app-89812429d8b2#.jg2dyy30l)|[example](https://github.com/Akryum/frontpage-vue-app), status:dev
+
+# Validation
+
+- [vuelidate](https://github.com/monterail/vuelidate), vue:2.0, links:[demo](https://monterail.github.io/vuelidate/?ref=multiselect), status:dev
```
Comment (Akryum) [inline PACKAGES.md]: You may want to add the [doc link](https://monterail.github.io/vuelidate/#getting-started) as well before the demo link.

File: PACKAGES.md [hunk 2]
```diff
@@ -13,7 +13,12 @@
 # Ui Components
 
 - [v-tooltip](https://github.com/Akryum/vue-tooltip), vue:2.0, links:[demo](https://akryum.github.io/vue-tooltip/), status:dev
+- [vue-multiselect](https://github.com/monterail/vue-multiselect), vue:1.0|2.0, links:[demo](http://monterail.github.io/vue-multiselect/), status:stable
```
Comment (Akryum) [inline PACKAGES.md]: You may want to add the [doc link](http://monterail.github.io/vue-multiselect/#sub-getting-started) as well before the demo link.

**Comment by shentao** [comment]
Hey @Akryum, hope this is what you meant! :)

File: PACKAGES.md [hunk 3]
```diff
@@ -13,12 +13,12 @@
 # Ui Components
 
 - [v-tooltip](https://github.com/Akryum/vue-tooltip), vue:2.0, links:[demo](https://akryum.github.io/vue-tooltip/), status:dev
-- [vue-multiselect](https://github.com/monterail/vue-multiselect), vue:1.0|2.0, links:[demo](http://monterail.github.io/vue-multiselect/), status:stable
+- [vue-multiselect](https://github.com/monterail/vue-multiselect), vue:1.0|2.0, links:[doc](http://monterail.github.io/vue-multiselect/#sub-getting-started), [demo](http://monterail.github.io/vue-multiselect/), status:stable
```
Comment (Akryum) [inline PACKAGES.md]: The array syntax is `item1|item2|item3`, so it should be:

```
- [vue-multiselect](https://github.com/monterail/vue-multiselect), vue:1.0|2.0, links:[doc](http://monterail.github.io/vue-multiselect/#sub-getting-started)|[demo](http://monterail.github.io/vue-multiselect/), status:stable
```

File: PACKAGES.md [hunk 4]
```diff
@@ -13,12 +13,12 @@
...
 
 # Data
 
 - [vue-apollo](https://github.com/Akryum/vue-apollo/), vue:1.0|2.0, links:[howto](https://dev-blog.apollodata.com/use-apollo-in-your-vuejs-app-89812429d8b2#.jg2dyy30l)|[example](https://github.com/Akryum/frontpage-vue-app), status:dev
 
 # Validation
 
-- [vuelidate](https://github.com/monterail/vuelidate), vue:2.0, links:[demo](https://monterail.github.io/vuelidate/?ref=multiselect), status:dev
+- [vuelidate](https://github.com/monterail/vuelidate), vue:2.0, links:[doc](https://monterail.github.io/vuelidate/#getting-started), [demo](https://monterail.github.io/vuelidate/?ref=multiselect), status:dev
```
Comment (Akryum) [inline PACKAGES.md]: See other comment.

**Comment by shentao** [comment]
Thanks! Corrected

**Pull request merged**